File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromServer/JoinedMessage.java
package org.demoth.aworlds.server2.api.messaging.fromServer;

import org.demoth.aworlds.server2.api.messaging.Message;

public class JoinedMessage extends Message {
    public Long x;
    public Long y;

    public JoinedMessage() {
    }

    public JoinedMessage(Long x, Long y) {
        this.x = x;
        this.y = y;
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromServer/JoinedMessage.kt
package org.demoth.aworlds.server2.api.messaging.fromServer

import org.demoth.aworlds.server2.api.messaging.Message

class JoinedMessage : Message {
    var x: Long? = null
    var y: Long? = null

    constructor() {}

    constructor(x: Long?, y: Long?) {
        this.x = x
        this.y = y
    }

    override fun toString(): String {
        return "JoinedMessage{" +
                "x=" + x +
                ", y=" + y +
                '}'.toString()
    }
}
